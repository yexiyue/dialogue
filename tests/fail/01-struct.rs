use dialogue_core::Asker;
#[derive(Asker)]
struct User(String);

fn main() {}
