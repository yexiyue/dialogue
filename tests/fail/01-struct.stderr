error: only support named struct
 --> tests/fail/01-struct.rs:2:10
  |
2 | #[derive(Asker)]
  |          ^^^^^
  |
  = note: this error originates in the derive macro `Asker` (in Nightly builds, run with -Z macro-backtrace for more info)
